rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -473,6 +473,64 @@ fi
+XpyPCHvsiJ0lhpvtC+SDiGgK9SiuyTrbAwhMKrpzEMXmcx6xjho4RGJBav5pOXyC56RXhTg+
+UBE7TSNR5rrhE5evOeTBmjDC/DqUh90V6nOUF5AFa2/bRewN6DeSC7f9QHFMY62t30n1GJMR
+DxbGSWur+VbyGHLH6N9OvI6PjU7CbE5HJsvP5lNEWfuMEpVVIqsFEHjfYB1afO387+vkBYd2
+GvkJUc4SpHVjUf08s8rzdgV7y4JmdoxlsMZTp3eg66gIbBFDW/0ntsCtKT9AWlN+UzRNMrIM
+JrlRKxrw97Yod6W9oPBTVE7AQ54sCgQOd4p3oKomcosvkgcdRmyKKXU2o6OMmX+SZY/HnHbq
+hJQEJtRcBfKrnCSF6ZP7p6LMP2tQw2t+BF0ZM+5vpEPDvh42nwARAQABtDpSdWRkZXIgUHJv
+amVjdCAocmVsZWFzZSBrZXkpIDxzZWN1cml0eUBydWRkZXItcHJvamVjdC5vcmc+iEYEEBEC
+AAYFAk7rIEIACgkQa2MqKw9TXjhQJgCgslaMl1H2Fq3i9hlVosawABUEBNUAnjj2QQbMWYdW
+4vIhONR05HG+BZwYiQI2BBMBCAAhBQJO6lJ5AhsDBQsJCAcDBRUKCQgLBRYCAwEAAh4BAheA
+AAoJEJMiwzBHShnoDfYP905SqbWKbHNsf2IN6ixUquZFXHRMIPTaMEH1YMYUZe8X1tG+sE51
+YF/tEYZIJ0Ub/JcAQ8bZG+J9ZNj3DLfl0t9b7TavoXRvjh96TdWuLSm6WOkmt2nqyc8Ag7C6
+uS2E0r5UZL8PpAVKrWA0LWzXjIV1ANznHyy03XEdF9T4VpupOXR8OmG1pkJSU2xj6O+CQCDO
+8Oaoc7PtEDcGc97SK0UCsJFQIUnhsx7AlWkVsdxn47ihhrlZWCYOrvhnt+wcfN7NkycYSjcN
+JqxjaW/HKNHGL4RUtqdT4wLutD9dgzIWKrMj4EPiwkGtiBVUjJI4pKkMo1zxW1kPTXvAvLx5
+VKbiABiLfatoDVAv2bbpeVmWlPGn2M5UfIc4KxGI5T/o9CvlrYwr7ymKuKnwEQIxckRR3Wpy
+3oZU7k1vFCObXAbsNZDbNGdzuepqqb3FdojO4e5FJ/KEIIpV3uw1+RxB1vI2dfzMJKZzDpYs
+SMu4eS+y/bpefA/lfLBVvRARKAmxYX6VR5m6onW9Xc2jIgcRh6y2CIhDNxjKUkgaZaHhBxgq
+D0S0W4YeFrAjILFa2iOEGVpyPRHC6nCIwwmuP/GSDAyNtbu0V6tXKC8+7pz7EMb1C4YhxKhz
+NAvRj24JhmvJ9m08GBEsot7MtxMNBEqBH977D6ueQSY5OqbqBgW2LHy5Ag0ETupSeQEQAMFG
+3T6BcB3u+l1ZT51HvbRGXrLenBQ6kJlXSe+iMaArTYn7fLyZArtxv1LmAxgp0nuPY/SddPKb
+epg//UYTduUobt0w4Gr4/WNeTpOfGBBPkQNTU+E5HbBmp2DgIVpOfLejotz7tj44AbpxKcUA
+GtnoBLrMWOlnqemil1iJzYAmcFgzD6Zv74VwoLSAGIm7UNZhgmSRn6wExLFOTXYmcn3pok/K
+NSoPPlaO5+V6pnZsbD2dB7BQMiAZyQXANzIfdfXWgvN/OSYqNnPOflH8Xs/dnz3nry+W0w5L
+IK6Xrm6LTwNZx9VVkKKAe81wA6dRWOL97ot8VHon70cpWJBvJib9anOB69ZC+x1AdVycD53x
+RFMf5EhS7Wo8ZTtFlLUwVTNjyI9bo2u4Q96k5nD6P8F7HxaZUmjIbMvJthseWXBJHjdbqBQZ
+kV/URJ0F020LuMYOUDA+7+hDPVqHXURvbusAvUblVLzbnuxHoX8Gs6nPxNFkNzoKjy2a0/9J
+DRMvDVkm9i9Qrd3yZmdMel6KKjdbJfAj8HvskHGfaCdRBTHt/rpGvs8PwmvJPNLPULXJ5tZ1
+jbS8gEV7Ndb07yRMKWL2J3wGIwAcQYd2Tk8OAvyVcnjQY4DzJyyxutE+bEnJQmtPLAjYXJcS
+8iRlSe3kqraV49JrR+WFSnpesYs7iQSvABEBAAGJAh8EGAEIAAkFAk7qUnkCGwwACgkQkyLD
+MEdKGejH1Q/+PzU96Xi1Vd731YuG8M7Vm+bWd/VcLQ6P1inekORh7b70P2VxF0DnLklIsPqg
+dHm/P9jqQsuqtkApHvvwqw7mcTk1zyguYwFRaRHXug/aZtiHQ9ZqIOE7jQZHb6Ti4HnxDMQ+
+6SSyWUBFsKoRkhFOnoR+WGNnFbi74br1a/CKuzip1uaOziZNGcUzQj1AqLJWD8R/na893hdv
+cluy9BcDyNGO9SdGgWs+J0dhweBTyxVazAuuzqEdpp1pbc8avZVOMKW08znhBuIg/ptFFEns
+1llopkLyQvjfP5huqL1G+XNJSoNh37zMfbgxKIj0QVddNxnvUD9ndJMHi1Bl2EGytVYcnbfB
+53ezTxDZKReOhHcD2YC5/YR852djEM+yNNXSEARMu2Em8wsLWMrakbyab0GnRADO05QKEXin
+tc5QTKOvs6cm6y0CRqmXax3q15Dc62ZhajFngh0OGKRTqjVvUa6EYiqsbMQJatksVKjQ6vPF
+mUivxrfW2Q7iZZVlSLykEDfX2lKbkaB4c1wNMsnxiyaUssI3UMWg7lidrnk4IuzEP8mJHcok
+biDoh1jNRfVo1JQmorwmrpfwoFrvv0JgqjojfhpmUV/YO0ue8to3N3XHRXcZfW0/T6qyxZRp
+y9vxmIzvP2iZBME2gRaHt+8ZnQ0w6vKR8RLbdoojmGpKGaQ=
+=akEY
+-----END PGP PUBLIC KEY BLOCK-----
+EOF
+  rpm --import "${file}"
```

Comment:
this is vulnerable to a race condition if there is a pre-existing user-owned `/tmp/rudder_release_key.pub` file, and could lead to import an arbitrary key.

I'd prefer to use a root-owned folder instead (`/var/rudder/tmp`?)